sdk/search/azure-search-documents/samples/async_samples/sample_vector_search_async.py

```diff
@@ -179,7 +179,7 @@ async def simple_hybrid_search():
             vector_queries=[vector_query],
             select=["hotelId", "hotelName"],
         )
-        print(await results.get_answers())
```

Comment:
Deleted these as they output None, since the search parameters don't specify semantic ranker/answers